source/type/tests/comments_test.cpp

```diff
@@ -48,6 +48,22 @@ struct logger_initialized {
 };
 BOOST_GLOBAL_FIXTURE(logger_initialized);
 
+namespace navitia {
+namespace type {
+bool operator==(const std::vector<Comment>& lhs, const std::vector<std::string>& rhs) {
+    if (lhs.size() != rhs.size()) {
+        return false;
+    }
+    for (size_t i = 0; i < lhs.size(); ++i) {
+        if (lhs[i].value != rhs[i]) {
```

Comment:
```suggestion
        if (lhs[i].value != rhs[i].value) {
```
Then also a check on `type`?

Reply:
`rhs` is a `string`, I could upatde every test to check the `type` too, but I dn't really see the point.

Reply:
Ah, fair enough I thought it was a comment.